test/mibs.jl

```diff
@@ -1,155 +1,63 @@
+module TestJuMPInput
```

Comment:
You can run the test file in the REPL as follows

```julia
include("test/mibs.jl")
```

If you're making changes to the package code, either restart the REPL every time, or use Revise.jl:
```Julia
using Revise
include("test/mibs.jl")
```